src/_backend_agg.h

```diff
@@ -1124,7 +1124,7 @@ class QuadMeshGenerator
 
     inline size_t num_paths() const
     {
-        return m_meshWidth * m_meshHeight;
+        return (size_t) m_meshWidth * m_meshHeight;
```

Comment:
py_adaptors.h defines num_paths as Py_ssize_t, perhaps better for consistency, though anything will work, of course.